### PR TITLE
Add a launch and configuration file for writing a ROS map

### PR DESCRIPTION
**Pull request by guilhermelawless**
Related issue: #577 

The min/max range default to the same as the backback_2d examples; same thing for the URDF file.

However, the name of both the config file and the launch file are kept generic.

### CODE REVIEW

**Comment by guilhermelawless** [comment]
Signed CLA.

File: cartographer_ros/launch/assets_writer_ros_map.launch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!--
+  Copyright 2016 The Cartographer Authors
```
Comment (cschuet) [inline cartographer_ros/launch/assets_writer_ros_map.launch]: 2018

File: cartographer_ros/configuration_files/assets_writer_ros_map.lua [hunk 2]
```diff
@@ -0,0 +1,36 @@
+-- Copyright 2016 The Cartographer Authors
```
Comment (cschuet) [inline cartographer_ros/configuration_files/assets_writer_ros_map.lua]: 2018

**Comment by cschuet** [comment]
Thanks for the contribution. Just some nits.

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 2.

**Comment by wally-the-cartographer** [comment]
Error: PUT https://api.github.com/repos/googlecartographer/cartographer_ros/pulls/702/merge: 405 At least one approved review is required by reviewers with write access. []

Log:
~~~
=> handle work item: PR 702 on googlecartographer/cartographer_ros.
 => Switching to branch master.
 $ git reset --hard HEAD [data/cartographer_ros]
HEAD is now at 5cd81be Follow googlecartographer/cartographer#859 (#712)
 $ git clean -fd [data/cartographer_ros]
 $ git checkout master [data/cartographer_ros]
Already on 'master'
Your branch is up to date with 'origin/master'.
 $ git pull [data/cartographer_ros]
Already up to date.
 => Checking out guilhermelawless/ros-map-example as local branch pr_702.
 $ git remote add guilhermelawless git@github.com:guilhermelawless/cartographer_ros.git [data/cartographer_ros]
fatal: remote guilhermelawless already exists.
 $ git fetch guilhermelawless [data/cartographer_ros]
 $ git branch --track pr_702 guilhermelawless/ros-map-example [data/cartographer_ros]
Branch 'pr_702' set up to track remote branch 'ros-map-example' from 'guilhermelawless'.
 => Switching to branch pr_702.
 $ git reset --hard HEAD [data/cartographer_ros]
HEAD is now at 5cd81be Follow googlecartographer/cartographer#859 (#712)
 $ git clean -fd [data/cartographer_ros]
 $ git checkout pr_702 [data/cartographer_ros]
Switched to branch 'pr_702'
Your branch is up to date with 'guilhermelawless/ros-map-example'.
 $ git pull --force [data/cartographer_ros]
Already up to date.
 => merge master.
 $ git merge master [data/cartographer_ros]
Already up to date.
 => Running clang-format on all *.cc and *.h files
 $ git commit -am Ran clang-format. [data/cartographer_ros]
On branch pr_702
Your branch is up to date with 'guilhermelawless/ros-map-example'.

nothing to commit, working tree clean
 $ git push --force guilhermelawless HEAD:ros-map-example [data/cartographer_ros]
Everything up-to-date
 => Abandoning branch pr_702.
 => Switching to branch master.
 $ git reset --hard HEAD [data/cartographer_ros]
HEAD is now at 8242cae Merge branch 'master' into pr_702
 $ git clean -fd [data/cartographer_ros]
 $ git checkout master [data/cartographer_ros]
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
 $ git branch -D pr_702 [data/cartographer_ros]
Deleted branch pr_702 (was 8242cae).

~~~

**Pull request closed without merging**